### PR TITLE
Make sure to set the variationContext for sitemap

### PR DESCRIPTION
**Pull request by patrickdemooij9**
Fixes https://github.com/patrickdemooij9/SeoToolkit.Umbraco/issues/162
And should also fix the issues reported by @ianleeder in https://github.com/patrickdemooij9/SeoToolkit.Umbraco/issues/123

### CODE REVIEW

File: src/SeoToolkit.Umbraco.MetaFields.Core/Services/MetaFieldsValueService/MetaFieldsValueService.cs [hunk 1]
```diff
@@ -23,29 +23,30 @@ public MetaFieldsValueService(IMetaFieldsValueRepository repository, IVariationC
...
                 {
-                    _repository.Add(nodeId, key, GetCulture(), value);
+                    _repository.Add(nodeId, key, foundCulture, value);
                 }
             }
             ClearCache(nodeId);
         }
 
-        public void Delete(int nodeId, string fieldAlias)
+        public void Delete(int nodeId, string fieldAlias, string culture)
```
Comment (ianleeder) [inline src/SeoToolkit.Umbraco.MetaFields.Core/Services/MetaFieldsValueService/MetaFieldsValueService.cs]: @patrickdemooij9 Should this culture default to null, same as the interface?
```suggestion
        public void Delete(int nodeId, string fieldAlias, string culture = null)
```

**Review by ianleeder — COMMENTED** [review]
Just as an FYI, it looks like you've committed these changes with a mixture of tabs/spaces.  GitHub is showing tabs as 8 spaces so it's super-obvious in the web interface  
image

File: src/SeoToolkit.Umbraco.MetaFields.Core/Services/MetaFieldsValueService/MetaFieldsValueService.cs (hunk 1 above)
Comment (patrickdemooij9) [reply src/SeoToolkit.Umbraco.MetaFields.Core/Services/MetaFieldsValueService/MetaFieldsValueService.cs]: Good catch. Not quite sure why it wasn't complaining about it but I changed it to the default null now

**Comment by patrickdemooij9** [comment]
Thanks for the check on my code @ianleeder! Guess I still have to put some settings correct on my new laptop

**Comment by ianleeder** [comment]
Not a problem @patrickdemooij9.  Any estimates on when you'll release the next version?  My project is going live soon, and I'd like to slip this fix in if possible.

**Comment by patrickdemooij9** [comment]
I am going to try and get this bug fixed: https://github.com/patrickdemooij9/SeoToolkit.Umbraco/issues/164 and then I'll deploy a release. The new version will most likely be available before next saturday

**Comment by patrickdemooij9** [comment]
@ianleeder 
The newest version is now out. This includes this change, so your code should work without issues now

**Pull request merged**